src/components/PodcastCard/PodcastCard.js

```diff
@@ -1,21 +1,48 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
+import { useState } from 'react'
 
 PodcastCard.propTypes = {
   image: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
+  author: PropTypes.string,
+  description: PropTypes.string,
+  categories: PropTypes.object,
```

Comment:
missing `is.Required` on purpose?

Reply:
Yes because my process was, that the expandable content is not required. But i can see that it might be